src/app/teams/teams.service.ts

```diff
@@ -107,7 +107,10 @@ export class TeamsService {
         });
         return this.http.request(request).do(() => {
             this.toastyService.success('Join request sent!');
-        }).catch((err, caught) => {
+        }).catch((err) => {
+            if (err.status === 409) {
+                this.toastyService.warning('You are already in team');
```

Comment:
@mrlafranchi , any funny ideas about this one? It occures if you are trying to ask to join a team, that you're already in.